### ISSUE TITLE
Script: 032B (Create Weapon Pickup) is poorly stubbed

### ISSUE THREAD

**Issue by JayFoxRox** [rwengine/openrw#132]
```
bindUnimplemented( 0x032B, game_create_weapon_pickup, 7, "Create Weapon Pickup" );
```

Due to this code there is no exception when the opcode is called, however, this opcode expects a weapon pickup handle to be returned to args[6].globalInteger (which does not happen).
As no pickup is created the handle is likely invalid and will cause crashes in other opcodes (because of invalid pointers).

This should either be stubbed correctly OR this should be implemented.

(Note that we already support 0x0213 which can create weapons too?)

**Comment by JayFoxRox** [comment]
This makes it impossible to enable 03DD (which was also not tested because of that).
It's used in LUIGI4 (Pump Action Pimp)

**Comment by JayFoxRox** [comment]
This has been fixed by #147
